components/brave_vpn/connection/mac/BUILD.gn

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
this should move to `brave/components/brave_vpn/browser/connection/mac`

Reply:
removed, this has been moved already